core/src/main/java/org/apache/accumulo/core/client/mapred/AccumuloFileOutputFormat.java

```diff
@@ -62,7 +62,10 @@
    * @param job
    *          the Hadoop context for the configured job
    * @since 1.5.0
+   * @deprecated since 1.7.0 This method returns a type that is not part of the public API and is not guaranteed to be stable. The method was deprecated to
```

Comment:
Do we have a feeling for what a user might have been doing if they called this method and what is the recommended method now?